### ISSUE TITLE
Javascript should be linted

### ISSUE THREAD

**Issue by mdboom** [mreid-moz/jupyter-notebook-gist#63]
We should run `eslint` or similar over all of the Javascript files as part of Travis.

Any preferences or best practices we want to follow?  I have no preference other than just picking something.

**Comment by jezdez** [comment]
@spasovski @openjck Any good tips on this one?

**Comment by openjck** [comment]
Yes! ESLint has a ton of configuration options, so I'd recommend starting by extending [`eslint:recommended`](https://eslint.org/docs/user-guide/configuring#using-eslintrecommended) and then making additional tweaks as needed from there.
